graphql/version_resolver.go

```diff
@@ -509,10 +510,10 @@ func (r *versionResolver) VersionTiming(ctx context.Context, obj *restModel.APIV
 func (r *versionResolver) Warnings(ctx context.Context, obj *restModel.APIVersion) ([]string, error) {
 	v, err := model.VersionFindOneId(utility.FromStringPtr(obj.Id))
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding version with id `%s`: %s", *obj.Id, err.Error()))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding version with id `%s`: %s", utility.FromStringPtr(obj.Id), err.Error()))
 	}
 	if v == nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding version with id `%s`: %s", *obj.Id, "version not found"))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding version with id `%s`: %s", utility.FromStringPtr(obj.Id), "version not found"))
```

Comment:
can omit the extra error message since it doesn't add any extra info
```suggestion
		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("version '%s' not found", utility.FromStringPtr(obj.Id)))
```